presto-jdbc/pom.xml

```diff
@@ -80,8 +80,24 @@
             <artifactId>guava</artifactId>
             <exclusions>
                 <exclusion>
-                    <groupId>*</groupId>
-                    <artifactId>*</artifactId>
+                    <groupId>com.google.errorprone</groupId>
```

Comment:
I believe this is wrong and is causing shading issues in presto-jdbc: org/checkerframework is now creeping into presto-jdbc jar. The groupId should be com.google.errorprone for the checker-qual

Please fix asap.

Reply:
I am fixing this in https://github.com/prestodb/presto/pull/23304

Reply:
Thanks, we assume there is no way to provide a test to avoid this issue, is there?

Reply:
Thanks, we assume there is no way to provide a test to avoid this issue, is there?